README.md

```diff
@@ -27,6 +27,22 @@ Install the latest version of `confluent` to `/usr/local/bin`:
 
     brew install confluentinc/tap/cli
 
+#### YUM (RHEL and CentOS)
+
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `2.27` or above for `arm64`):
+
+    sudo rpm --import https://packages.confluent.io/confluent-cli/rpm/archive.key
+    sudo yum-config-manager --add-repo https://packages.confluent.io/confluent-cli/rpm/confluent-cli.repo
+    sudo yum clean all && sudo yum install confluent-cli
+
+#### APT (Ubuntu and Debian)
+
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `2.27` or above for `arm64`):
+
+    wget -qO - https://packages.confluent.io/confluent-cli/deb/archive.key | sudo apt-key add -
```

Comment:
What does the dash after `-q0` do?

Reply:
The `-O` flag prints the result to file, but if the filename is `-` it prints to standard output instead. That allows us to pipe it to apt-key.